flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/FlinkSqlParserImplTest.java

```diff
@@ -362,19 +362,19 @@ public void testCreateTableWithWatermarkStrategyFromSource() {
 	}
 
 	@Test
-	public void testCreateTableWithComplexType() {
+	public void testCreateTableWithDecimalType() {
```

Comment:
Shall we drop this test. Just testing decimal seems not useful given we have a `testCreateTableWithComplexType` already?

Reply:
Removed.